app/kontainerdriver_data.go

```diff
@@ -88,6 +88,16 @@ func addKontainerDrivers(management *config.ManagementContext) error {
 	); err != nil {
 		return err
 	}
+	if err := creator.addCustomDriver(
+		"oraclecontainerengine",
+		"https://github.com/rancher-plugins/kontainer-engine-driver-oke/releases/download/v1.3.0/kontainer-engine-driver-oke-linux",
+		"",
+		"https://cdn.jsdelivr.net/gh/rancher-plugins/ui-cluster-driver-oke@v1.3.2/dist/component.js",
+		false,
+		"*.jsdelivr.net",
```

Comment:
Is this an oracle controller domain? The whitelist is to ensure we don't download random content from the internet and attempt to run it. If this is oracle controlled then it can be updated to `cdn.jsdelivr.net` and if not then it needs to be moved somewhere oracle controlled.

Reply:
Spoke with @vincent99 this morning about this. We decided to avoid the need of a CDN by moving the UI component into the [Rancher UI repository](https://github.com/rancher/ui/).

Reply:
checksum added